upload.py

```
@@ -1804,27 +1804,27 @@ async def process_meta(meta: Meta, base_dir: str) -> bool:
 
-                            parsed_url = urllib.parse.urlparse(poster_url)
+                            parsed_url = urllib.parse.urlparse(artwork_url)
                             if parsed_url.scheme in ("http", "https"):
                                 Path(poster_jpg_path).parent.mkdir(parents=True, exist_ok=True)
-                                urllib.request.urlretrieve(poster_url, poster_jpg_path)  # noqa: S310
-                                cover_path = poster_jpg_path
-                                meta.cover_path = cover_path
+                                urllib.request.urlretrieve(artwork_url, poster_jpg_path)  # noqa: S310
+                                artwork_path = poster_jpg_path
+                                meta.artwork_path = artwork_path
                         except Exception as e:
```

Ruff (0.16.0)

[warning] Do not catch blind exception: `Exception`

(BLE001)